com.cburch.Logisim.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.cburch.Logisim</id>
+  <launchable type="desktop-id">com.cburch.Logisim.desktop</launchable>
+  <name>Logisim</name>
+  <developer_name>carlburch</developer_name>
+  <summary>Logisim a graphical tool for desigining and simulating logic circuits</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  <url type="homepage">http://www.cburch.com/logisim/index.html</url>
+
+  <categories>
+    <category>education</category>
+  </categories>
+
+  <description>
+    <p>Logisim is an educational tool for designing and simulating digital logic circuits. With its
+      simple toolbar interface and simulation of circuits as you build them, it is simple enough to
+      facilitate learning the most basic concepts related to logic circuits. With the capacity to
+      build larger circuits from smaller subcircuits, and to draw bundles of wires with a single
+      mouse drag, Logisim can be used (and is used) to design and simulate entire CPUs for
+      educational purposes.</p>
+    <p>Logisim is used by students at colleges and universities around the world in many types of
+      classes, ranging from a brief unit on logic in general-education computer science surveys, to
+      computer organization courses, to full-semester courses on computer architecture.</p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">https://i.imgur.com/Vgm01Ip.png</screenshot>
```

Comment:
imgur is not a reliable storage for screenshots.

Reply:
Hosted screenshots on GitHub.